core/src/components/cat-select/cat-select.tsx

```diff
@@ -65,6 +65,16 @@ export interface CatSelectState {
   totalElements?: number;
 }
 
+export interface CatSelectMultipleTaggingValue {
+  ids?: string[];
```

Comment:
make these non-optional and use an empty array if it is empty. same for the single select tagging value